package.json

```diff
@@ -79,7 +79,7 @@
     "jest-environment-jsdom": "29.7.0"
   },
   "engines": {
-    "node": "^14.18 || ^16.13 || >=18"
+    "node": ">=18"
   },
```

Comment:
I was about to add this, but can you left support for 16. this is the only version I can run local env

Reply:
It's time to upgrade @irenelagno 🙂

Reply:
@irenelagno are you saying it's not working for you on node 18?  It works for me on 18.13.0.